repos/system_upgrade/el7toel8/actors/networkmanagerreadconfig/tests/unit_test.py

```diff
@@ -0,0 +1,51 @@
+from leapp.libraries.actor import library
+from leapp.models import NetworkManagerConfig
+
+
+def test_nm_with_dchp(monkeypatch):
+    config = library.read_nm_config(file_path='tests/files/nm_cfg_with_dhcp')
+    parser = library.parser_nm_config(config)
+
+    assert config
+    assert parser
+    assert parser.has_option('main', 'dhcp')
+
+
+def test_nm_without_dchp(monkeypatch):
+    config = library.read_nm_config(file_path='tests/files/nm_cfg_without_dhcp')
+    parser = library.parser_nm_config(config)
+
+    assert config
+    assert parser
+    assert not parser.has_option('main', 'dhcp')
+
+
+def test_nm_with_error(monkeypatch):
+    config = library.read_nm_config(file_path='tests/files/nm_cfg_file_error')
+    parser = library.parser_nm_config(config)
+
+    assert config
+    assert parser
+    assert not parser.has_section('main')
+
+
+def test_actor_with_dhcp(monkeypatch):
```

Comment:
This test seems superfluous to me. Almost everything is covered by previous tests. If you want to test actor execution, you shold move the test rom `unit_test.py` to `component_test.py` and use `current_actor_context` fixture, see RTD for the [fixture](https://leapp.readthedocs.io/en/latest/pydoc/leapp.html#leapp.snactor.fixture.current_actor_context) and [component tests](https://leapp.readthedocs.io/en/latest/unit-testing.html#writing-tests-that-execute-the-whole-actor-component-tests) and `component_test.py` of other actors for inspiration.